packages/perseus-core/src/utils/util.ts

```diff
@@ -21,6 +21,54 @@ const seededRNG: (seed: number) => RNG = function (seed: number): RNG {
     };
 };
 
+// Note: Previously called cyrb128
+function getHashes(str: string): [number, number, number, number] {
```

Comment:
```suggestion
function getHash(str: string): [number, number, number, number] {
```

I think this function should be called `getHash`. Semantically, it returns a _single_ 128-bit hash, but since JavaScript doesn't have a 128-bit number type, the value is split into four 32-bit integers.

Reply:
Updated! :)